README.md

```diff
@@ -1,24 +1,29 @@
 # HIPIFY
+HIPIFY is a set of tools to translate CUDA source code into portable [HIP][2]  C++ automatically.
 
-### Tools to translate CUDA source code into portable HIP C++ automatically
-## Table of Contents
+# rocBLAS
+rocBLAS is the AMD library for Basic Linear Algebra Subprograms (BLAS) on the [ROCm platform][1]. It is
+implemented in the [HIP programming language][2] and optimized for AMD GPUs.
 
-<!-- toc -->
+## Documentation
+Information about HIPIFY and other user topics can be found in the
+[HIPIFY documentation][3].
 
-- [hipify-clang](#clang)
-     * [Dependencies](#dependencies)
-     * [Usage](#hipify-clang-usage)
-     * [Building](#building)
-     * [Testing](#testing)
-     * [Linux](#linux)
-     * [Windows](#windows)
-- [hipify-perl](#perl)
-     * [Usage](#hipify-perl-usage)
-     * [Building](#hipify-perl-building)
-- [Supported CUDA APIs](#cuda-apis)
-- [Disclaimer](#disclaimer)
```

Comment:
The documentation has been reorganized such that the readme is not the starting point for documentation navigation. Users are directed to the website. The toc is handled there.

Reply:
The majority of the users are still looking at README.md which should have its own tocstop. Here is the biweekly stats:

![image](https://user-images.githubusercontent.com/10241941/210427192-435769d8-27c3-44bc-816f-ba56d438ff1b.png)

Reply:
I added a link from the README into the documentation source at index.md. I am hoping that will be a compromise on removing the TOC. 
The github repository will link to the documentation site by configuring the information below.
![image](https://user-images.githubusercontent.com/44449863/210449985-4775f2b2-e609-4e02-9ecf-75e89b1a6a94.png)
Overtime, the documentation portal will feature as high in the search engines than github repository. The intention is too direct users to our documentation portal to consume our documents. The documentation portal will contain links to the source repository and enable users to look at the doc source on github. Github will always remain the source of truth.

Reply:
> Overtime, the documentation portal will feature as high in the search engines than github repository.

I don't think so; I don't believe it'll ever happen. But it doesn't matter, as `README.md` is left alive in its place with your changes.

> Github will always remain the source of truth.

I agree it should be so, but I see that with the change we'll have four md files instead of a single `README.md`:
- A bit changed `README.md` - *Do not see any need for changing (especially stripping) it at all if it is for GitHub repo only.
- `hipify-clang.md`
- `hipify-perl.md`
- `supported_apis.md`

The information in the last three files is taken from `README.md` (duplicates).

Could you please explain to us how those last three are produced and how they are going to be supported?

Reply:
It looks like TOC in `README.md` can be left untouched without harm to the generated documentation. Correct me, if I'm wrong.

Reply:
Apologies, I restored the README.md when resolving merge conflicts, but this was not intentional. As the current README is relatively long, it seems easier to read on a site dedicated to documentation and link to the site in the README. The goal is to have the [Read the Docs site](https://advanced-micro-devices-demo--737.com.readthedocs.build/projects/HIPIFY/en/737/hipify-clang.html) replace the specific details in the README (see link for example).

If you agree, I propose the content below `<!-- tocstop -->` in the README be removed. This also prevents the problem you mentioned about having to support both the Read the Docs .md files and the README since it removes the duplicated content in the README.
This includes removing the hipify-clang/perl and supported CUDA API docs (migrated to Read the Docs), the disclaimer (generic/standard legal statements and licenses will be handled by rocm-docs-core in the future), and the TOC/table of contents (see the sidebar on the [RTD site](https://advanced-micro-devices-demo--737.com.readthedocs.build/projects/HIPIFY/en/737/supported_apis.html)).

Reply:
Hello, @samjwu,

> I restored the README.md when resolving merge conflicts, but this was not intentional

Does it mean the `README.md` won't be available in the repo?

>As the current README is relatively long, it seems easier to read on a site dedicated to documentation and link to the site in the README.

I don't think the relative length of the `README.md` is a strong reason here. More details - fewer mistakes - lesser issues like `doesn't work for me` reported, you know.

**Let me suggest yet another approach instead.**

1. We'll leave `README.md` almost untouched with its own TOC, but the contents will be constituted from `hipify-clang.md`, `hipify-perl.md`, and `supported_apis.md` if you wish so.
2. The TOC links will be changed to the corresponding `#` headers in those three files. Smth like follows:
```
- [hipify-clang](./docs/hipify-clang.md#clang)
     * [Dependencies](./docs/hipify-clang.md#dependencies)
     * [Usage](./docs/hipify-clang.md#hipify-clang-usage)
     * [Building](./docs/hipify-clang.md#building)
     * [Testing](./docs/hipify-clang.md#testing)
     * [Linux](./docs/hipify-clang.md#linux)
     * [Windows](./docs/hipify-clang.md#windows)
- [hipify-perl](./docs/hipify-perl.md#perl)
     * [Usage](./docs/hipify-perl.md#hipify-perl-usage)
     * [Building](./docs/hipify-perl.md#hipify-perl-building)
- [Supported CUDA APIs](./docs/supported_apis.md)
- [Disclaimer](#disclaimer)
```
3. The three child .md files should contain exact parts of the original `README.md` without any semantical or syntactical changes; the headers should be preserved as well.

Reply:
Works for me. I'll add back the TOC in the README and make those 3 .md files mentioned identical to the README for the relevant sections.

Reply:
[47e9926](https://github.com/ROCm-Developer-Tools/HIPIFY/pull/737/commits/47e9926058e7fe40d09fe4b083dff7b95296bcce) - implements points 1, 2, and 3

[c684bf8](https://github.com/ROCm-Developer-Tools/HIPIFY/pull/737/commits/c684bf8b46240827e5246399513416694802ca45) - updates links to docs in README and docs pages. also makes a couple changes to headers in docs pages for the RTD website links to work correctly

Reply:
Hi @emankov,

There needs to be some changes to the content on your README.md. @samjwu reverted many of my changes. They need to be restored. The new files created in docs/ are intended to move the contents of your README to a new location. The README in this project needs adopt README standards according to GitHub's README guidelines at https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-readmes. Additional requirements to the README structure may be mandated by ROCm's documentation team to standardize the look and feel. The README is not a single file documentation for the entire project. It is intended to point people to the documentation and other resources, if any, that can help the user understand how to use HIPIFY. HIPIFY documentation will be hosted online on a webserver. The webserver based documentation should be the first recommended option for users to consume documentation. The README may also link to the markdown files in the repository.

Reply:
Hello @saadrahim,

I didn't ask to revert; I just suggested yet another approach, and I do not see contradictions between it and your post. Can we discuss it offline? We have a corresponding thread.

Regards,
Evgeny